_cffi_build/seclink.py

```diff
@@ -0,0 +1,72 @@
+import os.path
+
+from cffi import FFI
+
+ffibuilder = FFI()
+
+_path = os.path.dirname(__file__)
+sourcefile = os.path.join(_path, 'seclink.cc')
+
+with open(sourcefile, 'r') as f:
+    source = f.read()
+
+
+ffibuilder.set_source(
+    "_truc",
```

Comment:
Where did this name come from? Maybe `_libseclink`?

Reply:
As it was a WIP, mainly to save the changes, I still had the WIP names. "truc" mean "thing" in French. I could have used foo instead, but old habits. `_libseclink` sounds more sensible.